src/components/input/index.jsx

```diff
@@ -112,6 +112,9 @@ class Input extends React.Component {
             label,
             name,
             maxLength,
+            numMax,
+            numMin,
+            numStep,
```

Comment:
thought you might want to be more verbose in case you don't realize what it is at first glance